src/models/vision_transformer.py

```diff
@@ -35,13 +37,14 @@ def __init__(
         qk_scale=None,
         drop_rate=0.0,
         attn_drop_rate=0.0,
-        norm_layer=nn.LayerNorm,
```

Comment:
The reason I'm removing this is because all attributes get serialized as JSON, similar to models in the Transformers library: https://huggingface.co/nielsr/vit-large-patch16-v-jepa/blob/main/config.json.

Reply:
Non-serializable attributes are omitted when building the `config.json` file.

This is double-edged: it means you don't have to remove it when integrating with `PyTorchModelHubMixin` but on the other hand if a user passes a custom value here and save the model on the Hub, other users that will reload the model won't have the exact same model.

Reply:
Ok, haved updated it to default to `partial(nn.LayerNorm, eps=1e-6)`